gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/BigQuery.java

```diff
@@ -275,8 +275,8 @@ private TableOption(BigQueryRpc.Option option, Object value) {
     /**
      * Returns an option to specify the table's fields to be returned by the RPC call. If this
      * option is not provided all table's fields are returned. {@code TableOption.fields} can be
-     * used to specify only the fields of interest. {@link BaseTableInfo#tableId()} and
-     * {@link BaseTableInfo#type()} are always returned, even if not specified.
+     * used to specify only the fields of interest. {@link TableInfo#tableId()} and
+     * {@link TableInfo#definition()} are always returned, even if not specified.
```

Comment:
I think if we say something like "and type, which is part of the {@link TableInfo#definition()}" are always returned.
So to not give the impression that the returned table definition will always be fully populated.